src/site/filters/liquid.js

```diff
@@ -349,28 +375,30 @@ module.exports = function registerFilters() {
     let deepObj = {};
 
     function findLink(arr, depth = 0) {
-      let d = depth;
-      // start depth at 1
-      d++;
-      for (const link of arr) {
-        // push the item into the trail
-        parentTree.push(link);
-
-        if (link.url.path === path) {
-          // we found the path! set 'found' to true and exit the recursion
-          deepObj = setDeepObj(parentTree, d, link);
-          found = true;
-          break;
-        } else if (link.links && link.links.length) {
-          // we didn't find it yet
-          // if the item has links, look for it within the links of this item (recursively)
-          findLink(link.links, d);
-          if (found) {
+      if (arr !== null) {
```

Comment:
My earlier comment still applies.

Reply:
I didn't write / change this filter function - the diff is a result of file formater changing the indenting. I'd rather not touch this, because I don't know what potential repercussions  may occur, @bkjohnson